testApp/src/Components/MovieDetail.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+    import { createEventDispatcher } from 'svelte';
+    import type { Movie } from '../Interfaces/Movie';
+    import { scale } from 'svelte/transition';
+
+    export let movie: Movie;
+    const dispatch = createEventDispatcher();
+
+    function backToGallery() {
+        dispatch('backToGallery');
+    }
+</script>
+
+<div class="movie-detail" on:click={backToGallery} in:scale={{ duration: 500 }} out:scale={{ duration: 500 }}>
+    <img src={movie.image} alt={movie.name} />
+    <h2>{movie.name}</h2>
+    <div class="additional-info">
+        <h2>Release Year: {movie.year}</h2>
+        <h2>Duration: {movie.runtime} minutes</h2>
```

Comment:
Is there no movie description available to show?

Reply:
After some research i found that the description was only given using a different api call.

Added the description:
https://github.com/BowenGeense/MovieShowcase/pull/3/files#diff-ef63d3406a68b222fc856f03cd2218f2a16eb0956f5261a5eb6cc1970f470bca